setup/tinyHIPPO_db_setup.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE alerts (
+	id                   integer NOT NULL  PRIMARY KEY,
+	alert_type           text CHECK (alert_type IN ('Privacy', 'IDS', 'System')) NOT NULL,
+	timestamp            text NOT NULL,
+	description          text NOT NULL,
+	severity             integer check (severity in (0, 1, 2)) NOT NULL,
+	mac_address          varchar(17),
```

Comment:
`NOT NULL`?

Reply:
Privacy alerts won't have a packet to pull addresses from

Reply:
^^ The mac_address cannot be a not null value